clang/test/BoundsSafety/CodeGen/unique-traps-O2.c

```diff
@@ -107,28 +91,39 @@ int consume(int* __bidi_indexable ptr, int* __bidi_indexable ptr2, int idx) {
 // CHECK-NEXT:    [[DOTNOT:%.*]] = icmp ugt ptr [[TMP1]], [[AGG_TEMP_SROA_2_0_COPYLOAD]], !annotation [[META7]]
 // CHECK-NEXT:    br i1 [[DOTNOT]], label %[[TRAP:.*]], label %[[CONT:.*]], !prof [[PROF8]], !annotation [[META7]]
 // CHECK:       [[TRAP]]:
-// CHECK-NEXT:    tail call void asm sideeffect "", "n"(i64 3) #[[ATTR3]], !annotation [[META7]]
-// CHECK-NEXT:    tail call void @llvm.ubsantrap(i8 25) #[[ATTR4]], !annotation [[META7]]
+// CHECK-NEXT:    tail call void @llvm.ubsantrap(i8 25) #[[ATTR3]], !annotation [[META7]]
 // CHECK-NEXT:    unreachable, !annotation [[META7]]
 // CHECK:       [[CONT]]:
 // CHECK-NEXT:    [[DOTNOT6:%.*]] = icmp ugt ptr [[ARRAYIDX]], [[TMP1]], !annotation [[META7]]
 // CHECK-NEXT:    br i1 [[DOTNOT6]], label %[[TRAP1:.*]], label %[[CONT2:.*]], !prof [[PROF8]], !annotation [[META7]]
 // CHECK:       [[TRAP1]]:
-// CHECK-NEXT:    tail call void asm sideeffect "", "n"(i64 4) #[[ATTR3]], !annotation [[META7]]
-// CHECK-NEXT:    tail call void @llvm.ubsantrap(i8 25) #[[ATTR4]], !annotation [[META7]]
+// CHECK-NEXT:    tail call void @llvm.ubsantrap(i8 25) #[[ATTR3]], !annotation [[META7]]
 // CHECK-NEXT:    unreachable, !annotation [[META7]]
 // CHECK:       [[CONT2]]:
 // CHECK-NEXT:    [[DOTNOT7:%.*]] = icmp ult ptr [[ARRAYIDX]], [[AGG_TEMP_SROA_3_0_COPYLOAD]], !annotation [[META9]]
 // CHECK-NEXT:    br i1 [[DOTNOT7]], label %[[TRAP3:.*]], label %[[CONT4:.*]], !prof [[PROF8]], !annotation [[META9]]
 // CHECK:       [[TRAP3]]:
-// CHECK-NEXT:    tail call void asm sideeffect "", "n"(i64 5) #[[ATTR3]], !annotation [[META9]]
-// CHECK-NEXT:    tail call void @llvm.ubsantrap(i8 25) #[[ATTR4]], !annotation [[META9]]
+// CHECK-NEXT:    tail call void @llvm.ubsantrap(i8 25) #[[ATTR3]], !annotation [[META9]]
 // CHECK-NEXT:    unreachable, !annotation [[META9]]
 // CHECK:       [[CONT4]]:
 // CHECK-NEXT:    [[TMP2:%.*]] = load i32, ptr [[ARRAYIDX]], align 4, !tbaa [[TBAA10]]
 // CHECK-NEXT:    ret i32 [[TMP2]]
 //
+int consume(int* __bidi_indexable ptr, int* __bidi_indexable ptr2, int idx) {
+  int other = i_want_to_be_inlined(ptr2, idx);
+  return ptr[idx];
+}
+//
```

Comment:
Are these extra comments emitted by UTC? Seems like a bug imo

Reply:
If you mean the ones after the closing brace of `consume` then I think so.